.github/actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -125,6 +126,14 @@ async function getLabelsToAdd( octokit, owner, repo, number ) {
 			if ( project.groups.ptype === 'github-actions' ) {
 				keywords.add( 'Actions' );
 			}
+
+			if ( project.groups.ptype === 'packages' ) {
+				keywords.add( '[Status] Needs Package Release' );
+			}
+			if ( project.groups.ptype === 'js-packages' ) {
+				keywords.add( '[Status] Needs JS Package Release' );
+				keywords.add( 'RNA' );
+			}
```

Comment:
We do not really use those labels anymore, now that we've automated most of the package. 

```suggestion
```

Reply:
Interesting, I didn't know that.
Removing 👍